### ISSUE TITLE
Not working on Android but working fine on IOS. 

### ISSUE THREAD

**Issue by MUZAFFAR-PURELOGICS** [NZME/react-native-ad-manager#138]
Ads are not working on the 0.72 version of react native project. It is showing blank space.
It is crashing by installing on 0.74 version of react native project.

**Comment by amitkumarsingh1991** [comment]
Yes, I am facing the same issue. Even I am using this version "react-native": "0.72.5". Is there anyone who can help us?

![image](https://github.com/user-attachments/assets/d486fd0d-d786-41f5-a315-d3ed54f5c0f6)

thanks in advance 👍

**Comment by amitkumarsingh1991** [comment]
I found the issue,

That issue due given method  here  adRequestBuilder.addNetworkExtrasBundle(AdMobAdapter.class, bundle); 
and that is inside this file node_modules/react-native-ad-manager/android/src/main/java/com/matejdr/admanager/BannerAdView.java. 


Solution : 
- Remove or comments out this function adRequestBuilder.addNetworkExtrasBundle(AdMobAdapter.class, bundle); because this function no more supported that why issue is there . 
- Make patch installation check here https://medium.com/@yusufsancakk/comprehensive-guide-to-patching-react-native-packages-f7f73836ce6c
